app/controllers/api/v3x0.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V3x0
+    class RootController < ApplicationController
+      def openapi
```

Comment:
@abellotti 

Is here reason to not leverage controllers from previous version like it is here 
https://github.com/lpichler/topological_inventory-api/blob/b1078f434b15e7db041721fd17805f85150d22e0/app/controllers/api/v2x0.rb#L10
?
and to have in v3.0 controllers  just `Mixins::IndexMixin` ?

thanks!

Reply:
@lpichler We need to pull in the `Api::V3x0::Mixins::IndexMixin`, not the one from `V1x0`

Reply:
👍 👍